src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -58,6 +57,8 @@ public function process(ContainerBuilder $container): void
                 $invokableRef = null;
             }
 
+            $definition->addTag('container.no_preload');
```

Comment:
See line 54, where the `$definition` variable is redefined as the new command service